src/main/resources/ccw-database-data.sql

```diff
@@ -70,15 +70,15 @@ VALUES
     ('DPF', 'Div pet fee','A')
 ;
 
-INSERT INTO CCW.CASE_STAGES(CASE_STAGE_ID)
+INSERT INTO CCW.CASE_STAGES(CASE_STAGE_ID, DESCRIPTION)
```

Comment:
Also, @Mapping(target = "description", defaultValue = "") needs to removed from CaseStagesMapper